.github/workflows/ci.yml

```diff
@@ -180,7 +278,11 @@ jobs:
         os="${{ matrix.os }}"
         cling_on=$(echo "${{ matrix.cling }}" | tr '[:lower:]' '[:upper:]')
         if [[ "${cling_on}" == "ON" ]]; then
-          git clone --depth=1 https://github.com/root-project/cling.git
+          #FIXME: Need latest version of cling not hardcoded.
+          git clone https://github.com/root-project/cling.git
+          cd ./cling 
+          git checkout tags/v1.0
```

Comment:
Instead of a hardcoded version of cling, we can add in the build matrix a new parameter for example cling-version:
`- name: ubu22-gcc9-clang13-cling
   os: ubuntu-22.04
   compiler: gcc-9
   clang-runtime: '13'
   cling: On
   cling-version: '1.0'
   ...`
And then this is written as:
`git checkout tags/v${{ matrix.cling-version }}`

Reply:
Done this change.